backend/server-connection/src/test/java/testutils/InMemoryIssueStore.java

```diff
@@ -44,7 +44,7 @@
 import static org.sonarsource.sonarlint.core.serverconnection.storage.StorageUtils.deserializeLanguages;
 
 public class InMemoryIssueStore implements ProjectServerIssueStore {
-  private final Map<String, Map<String, List<ServerIssue>>> issuesByFileByBranch = new HashMap<>();
+  private final Map<String, Map<String, List<ServerIssue<?>>>> issuesByFileByBranch = new HashMap<>();
```

Comment:
I saw some of those issues about ServerIssue being a generic class. I'm not sure it should be, but it's probably not related to your PR

Reply:
We have a rule in SQ to not return wildcards. But then we have other rules (or IDE) complaining about using types without a generic type. So this I decided to go in this direction, and we can mark issues as WF on SQ (or disable the rule).

Reply:
Ah yes, you are right. Let's keep that for later, as it might take some effort to refactor.